modules/nw-network-flows-object.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/tracking-network-flows.adoc
+
+[id="nw-network-flows-object_{context}"]
+= Network object configuration for tracking network flows
+
+The following YAML shows the configuration fields for network flows collectors in the Cluster Network Operator configuration.
+
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+  exportNetworkFlows: <.>
+    netFlow:
+      collectors:
+        - <netflow-collector-1>:2056
+        - <netflow-collector-2>:2056
+        ...
+    sFlow:
+      collectors:
+        - <sflow-collector-1>:6343
+        - <sflow-collector-2>:6343
+        ...
+    ipfix:
+      collectors:
+        - <ipfix-collector-1>:4739
+        - <ipfix-collector-2>:4739
+        ...
+----
+<.> The `exportNetworkFlows` field identifies the type of network flow collector, `netFlow`, `sFlow`, or `ipfix`, and the IP address and port for each collector. You can specify up to 10 collectors of each type.
+
+.Example configuration for tracking network flows
```

Comment:
Suggested edit: I would recommend putting `.Example configuration for tracking network flows` before the YAML example

Reply:
Agree 110%.  I cannot explain why it wasn't in that location when I submitted.